From 1167bd8747a5ec70a414e90393450aee82dcdb06 Mon Sep 17 00:00:00 2001
From: Liu Ying <victor.liu@nxp.com>
Date: Fri, 14 Sep 2018 10:51:03 +0800
Subject: [PATCH 4637/5242] MLK-19582 gpu: imx: imx8_dprc: Improve code for
 dprc_first_frame_handle()

commit  0e7ea0d55c45cead18a530bab339806d4d03cf1f from
https://source.codeaurora.org/external/imx/linux-imx.git

This patch contains code-change-only to improve code for
dprc_first_frame_handle().  No real logic modification.

Signed-off-by: Liu Ying <victor.liu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/gpu/imx/imx8_dprc.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/gpu/imx/imx8_dprc.c b/drivers/gpu/imx/imx8_dprc.c
index c5e7eba..0689cdd 100644
--- a/drivers/gpu/imx/imx8_dprc.c
+++ b/drivers/gpu/imx/imx8_dprc.c
@@ -591,8 +591,8 @@ void dprc_first_frame_handle(struct dprc *dprc)
 
 	if (dprc->is_blit_chan)
 		return;
-	else
-		dprc_write(dprc, REPEAT_EN, SYSTEM_CTRL0);
+
+	dprc_write(dprc, REPEAT_EN, SYSTEM_CTRL0);
 
 	prg_shadow_enable(dprc->prgs[0]);
 	if (dprc->use_aux_prg)
-- 
1.7.9.5

